operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -541,6 +541,14 @@ data:
               {{- end}}
             {{- end}}
             ]
+        - name: ISTIO_META_APP_CONTAINERS
```

Comment:
This was my concern with the snapshot...

Reply:
i updated the snapshot so that all the golden files in the webhook tests wouldn't have to be changed, because they were generated against a newer version of the charts. as long as we don't do that again (and this PR should prevent it) we should not ever have to touch those goldens unless the injection logic changes.